.github/workflows/building.yml

```diff
@@ -0,0 +1,47 @@
+name: Building
+
+on:
+  push:
+    branches: [ kawaii ]
```

Comment:
@acidicMercury8 is it possible to enable for all branches?

Reply:
@PeyTy this possibility exists, it is available using _filters_